lib/constants.js

```diff
@@ -1,20 +1,24 @@
 import CompoundPrizePoolBuilderRinkeby from '@pooltogether/pooltogether-contracts/deployments/rinkeby/CompoundPrizePoolBuilder.json'
 
-import StakePrizePoolBuilderMainnet from '@pooltogether/pooltogether-contracts/deployments/mainnet/StakePrizePoolBuilder.json'
-import StakePrizePoolBuilderRopsten from '@pooltogether/pooltogether-contracts/deployments/ropsten/StakePrizePoolBuilder.json'
+// import StakePrizePoolBuilderMainnet from '@pooltogether/pooltogether-contracts/deployments/mainnet/StakePrizePoolBuilder.json'
+// import StakePrizePoolBuilderRopsten from '@pooltogether/pooltogether-contracts/deployments/ropsten/StakePrizePoolBuilder.json'
 import StakePrizePoolBuilderRinkeby from '@pooltogether/pooltogether-contracts/deployments/rinkeby/StakePrizePoolBuilder.json'
 
-import SingleRandomWinnerBuilderMainnet from '@pooltogether/pooltogether-contracts/deployments/mainnet/SingleRandomWinnerBuilder.json'
-import SingleRandomWinnerBuilderRopsten from '@pooltogether/pooltogether-contracts/deployments/ropsten/SingleRandomWinnerBuilder.json'
-import SingleRandomWinnerBuilderRinkeby from '@pooltogether/pooltogether-contracts/deployments/rinkeby/SingleRandomWinnerBuilder.json'
+// import PoolWithMultipleWinnersBuilderMainnet from '@pooltogether/pooltogether-contracts/deployments/mainnet/PoolWithMultipleWinnersBuilder.json'
+// import PoolWithMultipleWinnersBuilderRopsten from '@pooltogether/pooltogether-contracts/deployments/ropsten/PoolWithMultipleWinnersBuilder.json'
+import PoolWithMultipleWinnersBuilderRinkeby from '@pooltogether/pooltogether-contracts/deployments/rinkeby/PoolWithMultipleWinnersBuilder.json'
 
-import RNGBlockhashMainnet from '@pooltogether/pooltogether-rng-contracts/deployments/mainnet/RNGBlockhash.json'
-import RNGBlockhashRopsten from '@pooltogether/pooltogether-rng-contracts/deployments/ropsten/RNGBlockhash.json'
+// import MultipleWinnersBuilderMainnet from '@pooltogether/pooltogether-contracts/deployments/mainnet/MultipleWinnersBuilder.json'
+// import MultipleWinnersBuilderRopsten from '@pooltogether/pooltogether-contracts/deployments/ropsten/MultipleWinnersBuilder.json'
+import MultipleWinnersBuilderRinkeby from '@pooltogether/pooltogether-contracts/deployments/rinkeby/MultipleWinnersBuilder.json'
+
+// import RNGBlockhashMainnet from '@pooltogether/pooltogether-rng-contracts/deployments/mainnet/RNGBlockhash.json'
+// import RNGBlockhashRopsten from '@pooltogether/pooltogether-rng-contracts/deployments/ropsten/RNGBlockhash.json'
 import RNGBlockhashRinkeby from '@pooltogether/pooltogether-rng-contracts/deployments/rinkeby/RNGBlockhash.json'
 
-import RNGChainlinkMainnet from '@pooltogether/pooltogether-rng-contracts/deployments/mainnet/RNGChainlink.json'
+// import RNGChainlinkMainnet from '@pooltogether/pooltogether-rng-contracts/deployments/mainnet/RNGChainlink.json'
 import RNGChainlinkRinkeby from '@pooltogether/pooltogether-rng-contracts/deployments/rinkeby/RNGChainlink.json'
 
 export const TICKET_DECIMALS = '18'
```

Comment:
Hard-coding this is a problem.  

If the user enters USDC, which uses 6 decimals, then tickets will be minted 1:1 but when displayed it'll appear as if the user has 0.0000001 tickets.

Ideally this can be pulled from the deposit type; if it's a cDai pool that would be Dai, or in a staking pool it would be the token entered.

Not all tokens have the 'decimals' function though, so otherwise we could default to 18